src/emulate.c

```diff
@@ -1288,6 +1289,7 @@ static void block_translate(riscv_t *rv, block_t *block)
         if (insn_is_branch(ir->opcode))
             break;
     }
+    (block->ir + block->n_insn - 1)->is_tail = true;
```

Comment:
Rewrite as `block->ir[block->n_insn - 1].is_tail = true;` which is shorter and better for bounds checks.